### PR TITLE
python-lsp-server: new port

### PR DESCRIPTION
**Pull request by gpanders**
#### Description

The py-language-server port is deprecated as the upstream project is
unmaintained. This port provides the maintained, community fork.

See https://github.com/palantir/python-language-server/issues/935.

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.15.7 19H1217 x86_64
Xcode 12.4 12D4e

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by gpanders** [comment]
@reneeotten Would you like to be added as a maintainer to the new port?

**Comment by reneeotten** [comment]
@gpanders - thanks for the PR! I would certainly like to stay the maintainer of these ports since they are dependencies of `py-spyder`. Setting the "original" ports to being "depracated with no upstream supports" is fine, once the Spyder project has released a version where they changed to the new dependencies, I will consider just removing the ports as nothing else depends on them. 

A few request still: 
- can you please add the `post-destroot` parts in the new submissions, I would like to include the license file, readme and other relevant files that are part of it
- please separate the commits such that each commit touches only one port

I'm a bit pre-occupied with familiy business right now so cannot promise that I will have time to follow-up on this soonish, but if you're willing to make the changes suggested above anyone with commit access should feel free to push the button!

**Comment by gpanders** [comment]
> A few request still:
> 
>     * can you please add the `post-destroot` parts in the new submissions, I would like to include the license file, readme and other relevant files that are part of it
> 
>     * please separate the commits such that each commit touches only one port

Done. Also, unless you object I just listed you as the sole maintainer on both `py-python-lsp-server` and `py-python-lsp-jsonrpc`.

File: python/py-python-lsp-server/Portfile [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+    depends_run-append \
+                    port:python-lsp-server_select
+
+    select.group    python-lsp-server
+    select.file     ${filespath}/py${python.version}-pylsp
+
+    notes "
+To make the Python ${python.branch} version of python-lsp-server the one \
+that is run when you execute the commands without a version suffix, e.g. \
```
Comment (ryandesign) [inline python/py-python-lsp-server/Portfile:81]: Avoid double spaces in notes output:
```suggestion
To make the Python ${python.branch} version of python-lsp-server the one\
that is run when you execute the commands without a version suffix, e.g.\
```

**Pull request merged**